s4e-web/src/app/views/map-view/view-manager/overlay-list-modal/overlay-list-modal.model.ts

```diff
@@ -0,0 +1,11 @@
+import {Modal} from '../../../../modal/state/modal.model';
+
+export const OVERLAY_LIST_MODAL_ID = 'overlay-list-modal-id';
+
+export interface OverlayListModal extends Modal {
```

Comment:
Why you are using an empty interface instead of its parent directly?

Reply:
It will be expanded when two types of Overlay Modal will be introduced (different style for institutions and personal settings)